src/api/README.md

```diff
@@ -28,6 +28,7 @@ An index file contains a YAML header with meta data describing the publication a
 | title | string | Yes | The title of the publication. |
 | subtitle | string | No | An optional subtitle. |
 | restricted | boolean | No | (default: `true`) Determines whether the publication is restricted to registered users or publically available. |
+| index | boolean | No | (default: `false`) Determines whether the publication is an index. |
```

Comment:
I thought about it, but I didn't about using the body as a description since the docs we have says the index file `contains no markdown content`, I was just applying the rules 😁.

However I have a reason for this, I wanted from `validateDocumentPayload` to do exactly what it says, and that means with more features/content we can manage them all in that function with no need for a filename.